sqs/queue_template_test.go

```diff
@@ -32,18 +32,41 @@ var _ = Describe("QueueTemplateBuilder", func() {
 		Expect(ok).To(BeTrue())
 	})
 
-	Context("when QueueName is set", func() {
+	Context("when QueueName is set for a non-FIFO queue", func() {
 		BeforeEach(func() {
 			builder.QueueName = "q-name-a"
 		})
-		It("should set primary queue name", func() {
+		It("should set primary queue name with a .stnd extension", func() {
```

Comment:
```suggestion
		It("should set primary queue name without a .fifo extension", func() {
```

Reply:
and similarly below

Reply:
ah thanks ... that was an abandoned path